### DISCUSSION TITLE
How to Start Quering only after Container has started!

### DISCUSSION THREAD

**Discussion by bill210kouk** [jertel/elastalert2#1257] [Q&A] [answered]
Hello!!!

I using Elastalert through a container. In general my rules are working. Recently i stopped the container for sometime around a day to be more specific. During Start of the container Load was created causing stress to my machine. I would like to know how can i set querying time to start on the time the container is started. 

The question is how i can set permanently the container to start querying against elasticsearch only after the container has been started?
 
P.S I know that i can Use --start NOW as an argument inside the container.

Thanks you!

**Comment by jertel** [comment, marked as answer]
You will need to use that `--start NOW` parameter to override the start time. You are welcome to submit a PR to have it also read that parameter from a config file. The Contribution Guidelines document in the root of this project need to be followed, but it's mainly just making sure the docs and unit tests get updated.
